barberia/src/components/buttons.css

```diff
@@ -0,0 +1,25 @@
+.hot-shit-button{
+/* position: absolute; */
+width: 232px;
```

Comment:
Please can you remove commented out code? Also Prettier 💅

Reply:
done